### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by staviq** [Chia-Network/chia-blockchain#5257] [Ideas]
I'm not sure how to call this, but here's the idea:

Many many people are discouraged by the low win chance and they are afraid their setup isn't working properly. Forums threads are floded with people questioning their setup because they haven't won yet.

I know this is far fetched idea, but i propose that in addition to normal winnings, network members would be able to win minimal possible mojo amount every about 24h per say, 10 plots ?, for debuging and feedback purposes.

Amount does not matter, the idea is about having a clear confirmation that the setup is working and people are indeed, farming correctly.

**Comment by SackofFluor** [comment]
I really like the idea, even if I am not sure how an actual implementation would look like.

Because that was exactly my problem after around 50TiB of Plots and keeping my "estimated time to win" for 1 month at 15-17 days, I also started to wonder whether my system for performing at all.
Of course I did all the necessary checks as far as I could, yes my verification times weren't awesome but well below the required <5sec., yet I didn't win anything.

First I found a faucet that transferred like 0,0000001 XCH so I knew  my wallet was fine...

And then yesterday morning : "BOOM" my first two XCH!
So yes, I know how it feels if you are not sure whether your system performs or not and I think your idea is good.

Maybe it could be connected to something like at least 1 plot per day or something, to prevent people from just having 50 instances running on VMs or whatsoever to exploit it....
